jetty-keystore/pom.xml

```diff
@@ -13,7 +13,7 @@
 
   <properties>
     <bundle-symbolic-name>${project.groupId}.keystore</bundle-symbolic-name>
-    <bouncycastle.version>1.69</bouncycastle.version>
+    <bouncycastle.version>1.62</bouncycastle.version>
```

Comment:
Don't we have all the properties in the main POM?

Reply:
maybe we can move this one up. I'm not sure as it's used only here.
no real strong opinions. (for consistency main pom would be right).
I will change that

Reply:
No strong opinions here either.
Do what you feel makes the most sense.

Reply:
@olamy why have you downgraded the version from `1.69` to `1.62`.

Reply:
because it doesn't work with 1.69 in the mod file :) . By chance we have distribution test. before the dynamic stuff.
I will work on the upgrade in another PR.
Or maybe better with this? but I don't like mixing stuff.

Reply:
upgrade done as well